user_guide_src/source/libraries/images.rst

```diff
@@ -261,15 +261,15 @@ Adding a Text Watermark
 -----------------------
 
 You can overlay a text watermark onto the image very simply with the text() method. This is useful for placing copyright
-notices, photogropher names, or simply marking the images as a preview so they won't be used in other people's final
+notices, photographer names, or simply marking the images as a preview so they won't be used in other people's final
 products.
 
 ::
 
 	text(string $text, array $options = [])
 
 The first parameter is the string of text that you wish to display. The second parameter is an array of options
-that allow you to specify how the text should be displayed::
+that allows you to specify how the text should be displayed::
```

Comment:
plural is correct - subject is "options"

Reply:
means "allow" is correct?